### ISSUE TITLE
DeepSea (make) installation before Salt S/W install

### ISSUE THREAD

**Issue by noelmcloughlin** [SUSE/DeepSea#1200]
### Description of Issue/Question

`make install` fails with **Error 5** when `salt-master` and `salt-api` services are missing. This can occur when DeepSea is installed **before** configuring host as "Salt Master".
```
+	systemctl restart salt-master
+	systemctl restart salt-api
```
Instead of failure we could add conditional support for this use case. For example ...  _If services are not existing print a message reminding user to configure the host as Salt Master._  

This was discussed at https://github.com/SUSE/DeepSea/pull/1196

### Setup
(Please provide relevant configs and/or SLS files (Be sure to remove sensitive info).)

Fedora 27

### Steps to Reproduce Issue
(Include debug logs if possible and relevant.)

```
$ sudo make install 
...
...
systemctl restart salt-master
Failed to restart salt-master.service: Unit salt-master.service not found.
make: *** [Makefile:806: install] Error 5
```

**Comment by noelmcloughlin** [comment]
I'll raise a pr sometime for this.

**Comment by noelmcloughlin** [comment]
Resolved by #1285
